api/types/configs.go

```diff
@@ -0,0 +1,29 @@
+package types
+
+// configs holds structs used for internal communication between the
+// frontend (such as an http server) and the backend (such as the
+// docker daemon).
+
+import (
+	"github.com/docker/docker/runconfig"
+)
+
+// ContainerRmConfig holds arguments for the container remove
+// operation. This struct is used to tell the backend what operations
+// to perform.
```

Comment:
@cpuguy83 does this look better? Hopefully we're about to see all of these 'argument config structs' all in one place, so the context provided by them all in the same location should help as a form of implicit documentation.